.travis.yml

```diff
@@ -6,13 +6,14 @@ env:
   - distro: centos6
   - distro: ubuntu1604
   - distro: ubuntu1404
-  - distro: ubuntu1204
-  - distro: fedora24
+  - distro: fedora25
+  - distro: fedora26
   - distro: debian8
+  - distro: debian9
 
 script:
   # Download test shim.
-  - wget -O ${PWD}/tests/test.sh https://gist.githubusercontent.com/geerlingguy/73ef1e5ee45d8694570f334be385e181/raw/51ef4c5aea346b3f8dd2c2518233a70dac6407d9/ansible-role-test.sh
+  - wget -O ${PWD}/tests/test.sh https://gist.githubusercontent.com/lazzurs/5ab8703cd6cb57775001daea64b135a3/raw/c4420229859ff169bf5a82fa50aebeff320882e1/ansible-role-test.sh
```

Comment:
You have rollback this

Reply:
Once the above change has happened this test script won't need to change in this PR.